Automattic/wp-calypso
# Hosting Command Palette: Fix search case sensitivity

## Proposed Changes

This PR removes the case sensitivity from the search to ensure that the search gets the expected match regardless of whether the user input is case-sensitive. 

**Before**

<img width="871" alt="Screenshot 2023-12-28 at 4 06 08 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/785a2571-5d2a-4c00-ae71-9a39ce01cf2b">

**After**

<img width="891" alt="Screenshot 2023-12-28 at 4 06 18 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/6ad8fbb1-de1a-48f1-a436-4cce011b3310">

## Testing Instructions

* Navigate to Calypso Live Link
* Press `cmd + k` on macOS or `ctrl + k` on Windows to open the command palette
* Type `MANAGE` in all capitals and confirm that the items with the keyword present in the `label` or `searchLabel` appear in search
* Type `manage` in all capitals and confirm that the items with the keyword present in the `label` or `searchLabel` appear in search
* Try different combinations of lower and upper case and confirm that the items with the keyword present in `label` or `searchLabel` appear in search

## Pre-merge Checklist
